stdlib/public/Concurrency/Actor.cpp

```diff
@@ -1294,127 +1407,58 @@ static void processDefaultActor(DefaultActorImpl *currentActor,
   // Leave the tracking info.
   trackingInfo.leave();
 
-  // If we still have an active actor, we should give it up.
-  if (threadIsRunningActor && currentActor) {
-    currentActor->giveUpThread(runner);
-  }
-
+  // We are eliminating a processing job for the original actor, release it.
   swift_release(actor);
```

Comment:
I'm actually about 80% certain that this is wrong to be here since we're only scheduling with inline jobs and we don't ever take extra ref counts on the actor since we don't create extra processing jobs.